requirements-test.txt

```diff
@@ -1,7 +1,8 @@
 coveralls
 responses
 freezegun
-openwisp-utils[qa]>=0.2.1
+openwisp-utils[qa]>=0.3.0
```

Comment:
Travis error related to the fact taht `openwisp-utils` is updated.
Please investigate and fix it! :smile:

Reply:
change `0.3.0` to `0.3.1`